src/main/java/com/ozonehis/eip/odooopenmrs/component/OdooEndpoint.java

```diff
@@ -0,0 +1,59 @@
+package com.ozonehis.eip.odooopenmrs.component;
+
+import com.ozonehis.eip.odooopenmrs.client.OdooClient;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.camel.*;
+import org.apache.camel.spi.Metadata;
+import org.apache.camel.spi.UriEndpoint;
+import org.apache.camel.spi.UriPath;
+import org.apache.camel.support.DefaultEndpoint;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Odoo component to integrate with Odoo XML RPC.
+ */
+@UriEndpoint(firstVersion = "1.0.0", scheme = "odoo", title = "Odoo", syntax = "odoo:method/model", producerOnly = true)
```

Comment:
Why not set the method as a header or param? I think having something like odoo:get/re.user is less intuitive.